LoopKitUI/Views/ChartTableViewCell.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ChartTableViewCell.swift
+//  Naterade
+//
+//  Created by Nathan Racklyeft on 2/19/16.
+//  Copyright © 2016 Nathan Racklyeft. All rights reserved.
+//
+
+import UIKit
+
+
+public final class ChartTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var chartContentView: ChartContainerView!
+
+    @IBOutlet weak var titleLabel: UILabel?
+
+    @IBOutlet weak var subtitleLabel: UILabel?
+
+    public override func prepareForReuse() {
+        super.prepareForReuse()
+
+        chartContentView.chartGenerator = nil
+    }
+
+    public func reloadChart() {
```

Comment:
All of these functions were created because `IBOutlets` shouldn't be accessible outside of their defining module